presidio-analyzer/Dockerfile.windows

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-windowsservercore
+FROM python:3.12-windowsservercore-ltsc2022
```

Comment:
Why did you choose this base image?

Reply:
Looking at this website there are only 2 official docker images:

[3.12.7-windowsservercore-ltsc2022, 3.12-windowsservercore-ltsc2022⁠](https://github.com/docker-library/python/blob/37a7bfdfa9947e8eaf650150a5df1729e7432599/3.12/windows/windowsservercore-ltsc2022/Dockerfile)

[3.12.7-windowsservercore-1809, 3.12-windowsservercore-1809⁠](https://github.com/docker-library/python/blob/37a7bfdfa9947e8eaf650150a5df1729e7432599/3.12/windows/windowsservercore-1809/Dockerfile)

https://hub.docker.com/_/python?tab=tags&page=1&name=3.8.2&ordering=-name

Reply:
Is there an official version for python 3.9? 3.12 is still in "bugfix" status: https://devguide.python.org/versions/

Reply:
As far as i see v3.12 is the first with official windows images. https://github.com/docker-library/python

Reply:
What about these? https://hub.docker.com/r/microsoft/windows-cssc-python